src/components/migrationToV3.stories.mdx

```diff
@@ -62,18 +66,21 @@ const MyApp = () => {
     const response = await fetch('/my-meetings-endpoint');
     const data = await response.json();
     const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
+    const logger = new ConsoleLogger('SDK', LogLevel.WARN);
```

Comment:
No need to store `logger` and `eventController` as variables as they are not referred in this example. Simplify the logic here to make the example straightforward.

```js
const meetingSessionConfiguration = new MeetingSessionConfiguration(data.Meeting, data.Attendee);
const options = {
  eventController: new DefaultEventController(
      meetingSessionConfiguration, 
      new ConsoleLogger('SDK', LogLevel.WARN)
      new NoOpEventReporter()
    );
}

await meetingManager.join(
        meetingSessionConfiguration,
        options,
      );
```